sound/soc/sof/sof-pci-dev.c

```diff
@@ -439,6 +439,8 @@ static const struct pci_device_id sof_pci_ids[] = {
 #if IS_ENABLED(CONFIG_SND_SOC_SOF_ELKHARTLAKE)
```

Comment:
@libinyang For commit message, all upper-case is usually interpreted as yelling, so unless it's about an acronym, better to avoid all-uppercase in normal text. So maybe:

ASoC: SOF: Intel: add new PCI ID for ElkhartLake

Reply:
Got it. Thanks.